packages/addons/service/emby/changelog.txt

```diff
@@ -1,4 +1,11 @@
+<<<<<<< HEAD
 114
+=======
+115:
+- Update to version 3.2.26.0
+
+114:
+>>>>>>> 35c307893... Emby: update to 3.2.26.0
```

Comment:
@Jdiesel87 this needs to be resolved. Git can be mind-bending at first (but we've all been there) so if there's something you need help with please drop into the #libreelec IRC channel on freenode - there's usually a couple of people with good git skills in the channel. My personal advice is to stop using GUI tools until you understand how the underlying commands work from the console. There is no rush, and we prefer contributors to learn the right way to do things. Fixing this, rebasing the commit and force pushing change to the PR is quite simple and avoids having to close/reopen and create new PR's all the time. Thanks.